Pod/Classes/URBNAlertStyle.m

```diff
@@ -153,17 +153,21 @@ - (id)copyWithZone:(NSZone *)zone {
     // UIColors
     styler.buttonBackgroundColor = [[UIColor alloc] initWithCGColor:self.buttonBackgroundColor.CGColor];
     styler.destructionButtonBackgroundColor = [[UIColor alloc] initWithCGColor:self.destructionButtonBackgroundColor.CGColor];
+    styler.destructiveButtonTitleColor = [[UIColor alloc] initWithCGColor:self.destructiveButtonTitleColor.CGColor];
```

Comment:
How come you’re using `initWithCGColor:`? Since `UIColor` is immutable, you shouldn’t worry about creating a copy like this.